### ISSUE TITLE
Support code block line numbering features in preview

### ISSUE THREAD

**Issue by bbringhu** [AdobeDocs/adobe-markdown-authoring#31]
We now support line numbering parameters in code blocks. It would be great if VS Code preview could support it. 

Here's the documentation:

### Syntax highlighting for code blocks

Experience League supports syntax highlighting for code blocks. Make sure that you specify a language such as `java` after the opening set of back ticks to make sure that the syntax is highlighted properly. For a list of valid languages, see [https://prismjs.com](https://prismjs.com/#supported-languages). If any languages are missing, please log a jira ticket.

### Line numbering in code blocks

Add `{line-numbers="true"}` after the language to enable line numbering.

Example with line numbers (&grave;&grave;&grave;`html {line-numbers="true"}`): 

```html {line-numbers="true"}
<!DOCTYPE html>
<html>
<body>

<h1>My First Heading</h1>
<p>My first paragraph.</p>

</body>
</html>
```

**Start numbering on line _**

Add `start-number="n"` after line number syntax to start the numbering on a number other than 1.

Example with new start line (&grave;&grave;&grave;`html {line-numbers="true" start-line="7"}`): 

```html {line-numbers="true" start-line="7"}
<!DOCTYPE html>
<html>
<body>

<h1>My First Heading</h1>
<p>My first paragraph.</p>
<p>My second paragraph.</p>

</body>
</html>
```

### Line highlighting in code blocks

Add `highlight="n"` after line number syntax to highlight rows within a code block. Specifying `11-13, 16` will highlight lines 11 through 13 and 16.

Example with line highlighting (&grave;&grave;&grave;`html {line-numbers="true" start-line="7" highlight="11-13, 16"}`): 

```html {line-numbers="true" start-line="7" highlight="11-13, 16"}
<!DOCTYPE html>
<html>
<body>

<h1>My First Heading</h1>
<p>My first paragraph.</p>
<p>My second paragraph.</p>

</body>
</html>
```

**Comment by geverit4** [comment]
I have the transform working for these, and it is generating the correct HTML output.  

```html
<code class="html"><span class="token doctype"><span class="token punctuation">&lt;!</span><span class="token doctype-tag">DOCTYPE</span> <span class="token name">html</span><span class="token punctuation">&gt;</span></span>
<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>html</span><span class="token punctuation">&gt;</span></span>
<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>body</span><span class="token punctuation">&gt;</span></span>

<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>h1</span><span class="token punctuation">&gt;</span></span>My First Heading<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>h1</span><span class="token punctuation">&gt;</span></span>
<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>p</span><span class="token punctuation">&gt;</span></span>My first paragraph.<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>p</span><span class="token punctuation">&gt;</span></span>

<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>body</span><span class="token punctuation">&gt;</span></span>
<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>html</span><span class="token punctuation">&gt;</span></span>
<span aria-hidden="true" class="line-numbers-rows"><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span></span></code>
```
So, the hard part is done - but just like #5 , the transform works but the styles are not being applied in the preview.  Once that is fixed, it should work the same as on the demo site.

**Comment by geverit4** [comment]
Well that was unexpectedly challenging. 

![Screenshot 2023-04-27 at 3 42 53 PM](https://user-images.githubusercontent.com/1187616/234974396-4199bda8-72f4-4b62-b389-03ea1b40f154.png)

**Comment by bbringhu** [comment]
You made it work though. Nice!

**Comment by geverit4** [comment]
It was a CSS change in prism-line-numbers.css.

![Screenshot 2023-05-19 at 5 04 22 PM](https://github.com/AdobeDocs/adobe-markdown-authoring/assets/1187616/5b72e829-5bb3-4115-8774-197355e3d778)

**Comment by geverit4** [comment]
![Screenshot 2023-05-19 at 5 05 28 PM](https://github.com/AdobeDocs/adobe-markdown-authoring/assets/1187616/4da4a2fd-7ca7-4934-ae2f-251630367c92)

**Comment by bbringhu** [comment]
Fixed in 2.0.7
